src/databaseService/index.js

```diff
@@ -65,6 +65,18 @@ export const queryEmailFromUsername = async (username, collectionPath = 'users')
   }
 }
 
+export const queryUsernameFromEmail = async (email, collectionPath = 'users') => {
+  try {
+    const querySnapshot = await firebase.firestore().collection(collectionPath).where('email', '==', email).get()
+    if (hasDocuments(querySnapshot)) {
+      const json = getFirstDocumentDataFromQuerySnapshot(querySnapshot)
+      return json && json.username
+    }
+  } catch (error) {
```

Comment:
Please add an empty line after variables declarations (`const querySnapshot`) and the next line. Also, add an empty line before `return` statements.